Consistency Improvement of Examples (#190)
emschwartz/interledger-rs: examples/README.md

1. Integrating Interledger Into Your App (Coming Soon!)

Have questions? Feel free to [open an issue](https://github.com/emschwartz/interledger-rs/issues/new) or ask a question [on the forum](https://forum.interledger.org/)!

## Running the Examples
Each example includes instructions for running it step-by-step in the README.

If you want to run all of the steps automatically, you can use the provided [`run-md.sh`](../scripts/run-md.sh) script to parse and execute the shell commands from the Markdown file:

```bash
# Under the example directory, for example, "simple"

> $ cat README.md | ../../scripts/run-md.sh

Reviewer: Maybe it would be even nicer to make it so you can call `../../scripts/run-md.sh README.md`. We could make it so the script checks for arguments passed to it and `cat`s the file if one was passed. If not, it would read from `stdin` instead

Author: Fixed.

1. Integrating Interledger Into Your App (Coming Soon!)

Have questions? Feel free to [open an issue](https://github.com/emschwartz/interledger-rs/issues/new) or ask a question [on the forum](https://forum.interledger.org/)!

## Running the Examples

> Each example includes instructions for running it step-by-step in the README.

Reviewer 2: Suggest updating the sentence to - "The README of each example provides step-by-step instructions on how to the example."

Reviewer: ```suggestion
The README of each example provides step-by-step instructions on how to run the example.
```

```bash
# Under the example directory, for example, "simple"
$ cat README.md | ../scripts/run-md.sh
```

> ## For those who add examples

Author: I'll move it later.

1. Integrating Interledger Into Your App (Coming Soon!)

Have questions? Feel free to [open an issue](https://github.com/emschwartz/interledger-rs/issues/new) or ask a question [on the forum](https://forum.interledger.org/)!

## Running Scripts Collectively
Although instruction scripts are shown on the example pages, you can also run the scripts collectively and automatically using [lit](https://github.com/vijithassar/lit). To run scripts, install `lit`:

```bash
# Under the directory you want to install lit.sh to

> $ curl https://raw.githubusercontent.com/vijithassar/lit/master/lit.sh > lit.sh && chmod +x lit.sh

Reviewer: Unless you see a benefit of using `lit`, I would have a slight preference for the solution from the [`mdlp` gist](https://gist.githubusercontent.com/trauber/4955706/raw/0d9e6261bf78da5e448bea383a8ad9d51548843c/mdlp.awk) instead because it's _so_ simple. I think `lit` is a bit overkill because it has functionality for reading in files and writing them to specific places, as opposed to just operating on piped data.

Instead of telling the user to "install" something, it would be more like "just run this simple command, which we've copied into a file called run-md.sh in the repo for your convenience".
